### ISSUE TITLE
Can't update to rails 4.2.0

### ISSUE THREAD

**Issue by bricesanchez** [refinery/refinerycms#2816]
```ruby
Fetching gem metadata from https://rubygems.org/........
Fetching additional metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "actionmailer":
  In Gemfile:
    refinerycms (>= 0) ruby depends on
      refinerycms-authentication (= 3.0.0) ruby depends on
        actionmailer (< 4.2, >= 4.0.2) ruby

    rails (= 4.2.0) ruby depends on
      actionmailer (4.2.0)
```

Could we allow rails 4.2.0 in `refinerycms-authentication` ?

**Comment by parndt** [comment]
Globalize 5.0.0 is released! Maybe it'll work better now?

**Comment by bricesanchez** [comment]
Awesome! Thank you @parndt, we can go forward!

**Comment by parndt** [comment]
Maybe, there's an issue with generating a dummy app

**Comment by bricesanchez** [comment]
Fixed #2889 :smiley:
